bentoctl/deployment_config.py

```diff
@@ -83,26 +74,15 @@ def from_file(cls, file_path: t.Union[str, Path]):
 
         return cls(config_dict)
 
-    def validate_operator_spec(self, operator_schema):
-        """
-        validate the schema using cerberus and show errors properly.
-        """
-        # cleanup operator_schema by removing 'help_message' field
-        operator_schema = remove_help_message(schema=operator_schema)
-        v = cerberus.Validator()
-        validated_spec = v.validated(self.operator_spec, schema=operator_schema)
-        if validated_spec is None:
-            raise InvalidDeploymentSpec(spec_errors=v.errors)
-
-        return validated_spec
-
     def save(self, save_path, filename="deployment_spec.yaml"):
```

Comment:
I guess now we will start calling the .yaml file `deployment_config.yaml` right?

Reply:
Maybe.  I am still figuring out what would be a good name for that.